corehq/ex-submodules/pillowtop/management/commands/create_checkpoints_for_merged_pillows.py

```diff
@@ -123,15 +143,15 @@ def handle(self, **options):
                     checkpoint.topic,
                     checkpoint.partition,
                     checkpoint.offset))
-            msg = "Creating checkpoints for" if check else "Checkpoints to be created for"
+            msg = "Creating checkpoints for" if skip_check else "Checkpoints to be created for"
```

Comment:
looks like `skip_check` is always True now?

Reply:
You can see like 119 passing a `False` value